pkgs/by-name/ll/llama-cpp/package.nix

```diff
@@ -17,15 +17,11 @@
 , openclSupport ? false
 , clblast
 
-, openblasSupport ? true
+, openblasSupport ? !rocmSupport && !cudaSupport
```

Comment:
I'm now not entirely sure it should be these two, since openblas support only conflicts with opencl support and rocm suport, not cuda right?

Reply:
Hmm, the assert will have to be updated too then.

I don't use cuda, so I can only test rocm, opencl, and openblas. I can make it just `!rocmSupport` and `!cudaSupport` can be added if a cuda user has an issue.